### PR TITLE
Trading Rewards Copy

### PR DESCRIPTION
**Pull request by mike-dydx**
[TRCL-3446 : add trading rewards localization strings](https://linear.app/dydx/issue/TRCL-3446/add-trading-rewards-localization-strings)

[Figma](https://www.figma.com/file/fMQodZKzn9B5aZTN5G0fLJ/dYdX-%E2%80%BA-Desktop?type=design&node-id=19925-59720&mode=design&t=sX9XWhldt1sBu1v9-4)

### CODE REVIEW

File: config/localization/en/app.json [hunk 1]
```diff
@@ -1405,6 +1414,11 @@
       "USERS_REFERRED": "Users Referred",
       "YOU_CAN_WIN_THESE": "You can win these in our {LEAGUES_LINK}"
     },
+    "TRADING_REWARDS" : {
+        "EVENT": "Event",
+        "FOR_TRADING": "For trading {PERIOD}",
+        "TRADING_REWARD": "Trading Reward"
+    },
```
Comment (aforaleka) [inline config/localization/en/app.json:1421]: where is this from?

**Review by aforaleka — APPROVED** [review]

File: config/localization/en/app.json [hunk 2]
```diff
@@ -1394,6 +1401,8 @@
       "POSTAL_CODE": "Postal Code",
       "PRINT_NAME": "Print Name",
       "REWARDS_BY_EPOCH": "Rewards by Epoch",
+      "REWARDS_LAST_7_DAYS": "Rewards last 7 days",
```
Comment (aforaleka) [inline config/localization/en/app.json:1404]: don't think we need this anymore -- should be This week or This week rewards

File: config/localization/en/app.json [hunk 3]
```diff
@@ -128,6 +131,7 @@
       "GO_TO_MARKET": "Go to market",
       "GOVERNANCE": "Governance",
       "HEDGIES": "Hedgies",
+      "HELP": "Help",
```
Comment (jaredvu) [inline config/localization/en/app.json:134]: This key exists already

File: config/localization/en/app.json (hunk 2 above)
Comment (mike-dydx) [reply config/localization/en/app.json]: addressed in bb53067d74d4b1b48541e5e26847c81341be3130

File: config/localization/en/app.json (hunk 3 above)
Comment (mike-dydx) [reply config/localization/en/app.json]: addressed in bb53067d74d4b1b48541e5e26847c81341be3130

File: config/localization/en/app.json [hunk 4]
```diff
@@ -676,7 +683,8 @@
     "LEGAL": {
       "ACADEMY_LINK_DESCRIPTION": "To access the dYdX Academy, you will leave the dYdX Trading Inc. website and join a website made available by dYdX Foundation, which is independent from and unaffiliated with dYdX Trading Inc. dYdX Trading Inc. is not responsible for any action taken or content on the third-party website.",
       "LEGAL": "Legal",
-      "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you will leave the dYdX Trading Inc. website and join a website made available by dYdX Foundation, which is independent from and unaffiliated with dYdX Trading Inc. dYdX Trading Inc. is not responsible for any action taken or content on the third-party website.",
+      "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you need to export your secret phrase to Keplr Wallet.",
```
Comment (aforaleka) [inline config/localization/en/app.json:686]: ah it looks like now it's a duplicate of the one inside TRADING_REWARDS? should we just keep one?

also where is this supposed to show up? i think these should be the most updated according to feedback by antonio:
image

File: config/localization/en/app.json [hunk 5]
```diff
@@ -1314,7 +1322,31 @@
       "TRADE_TO_WIN": "Trade to win",
       "TRADERS_JOINED": "Traders Joined",
       "USERS_REFERRED": "Users Referred",
-      "YOU_CAN_WIN_THESE": "You can win these in our {LEAGUES_LINK}"
+      "YOU_CAN_WIN_THESE": "You can win these in our {LEAGUES_LINK}",
+      "REWARD_HISTORY_TOOLTIP": "Time is displayed in Coordinated Universal Time (UTC)"
```
Comment (aforaleka) [inline config/localization/en/app.json:1326]: might want to include this in the tooltip strings instead

File: config/localization/en/app.json [hunk 6]
```diff
@@ -1314,7 +1322,31 @@
...
+    },
+    "TRADING_REWARDS" : {
+        "ABOUT": "About",
+        "EVENT": "Event",
+        "FOR_TRADING": "For trading {PERIOD}",
+        "FOR_V4":"{SUBJECT} for v4",
+        "GOVERNANCE": "Governance",
+        "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you need to export your secret phrase to Keplr Wallet.",
+        "LAUNCH_INCENTIVES":"Launch Incentives",
+        "LAUNCH_INCENTIVES_DESCRIPTION":"Each season, you can earn rewards for trading on dYdX Chain. Trade to earn your rightful spot on the leaderboard.",
```
Comment (aforaleka) [inline config/localization/en/app.json:1336]: this is the most updated according to legal
image

File: config/localization/en/app.json [hunk 7]
```diff
@@ -1314,7 +1322,31 @@
...
+        "SEASON_ID":"Season {SEASON_ID}",
+        "STAKING": "Staking",
+        "STAKING_LINK_DESCRIPTION": "To stake, you need to export your secret phrase to Keplr Wallet.",
+        "TRADING_REWARD": "Trading Reward",
+        "FAQ_WHO_IS_ELIGIBLE_QUESTION":"Who is eligible for trading rewards?",
+        "FAQ_WHO_IS_ELIGIBLE_ANSWER":"All traders are eligible for trading rewards.",
+        "FAQ_HOW_DO_TRADING_REWARDS_WORK_QUESTION":"How do trading rewards work?",
+        "FAQ_HOW_DO_TRADING_REWARDS_WORK_ANSWER":"Immediately after each fill, trading rewards are sent directly to the trader’s dYdX Chain address, based on the amount of fees paid by the trader.",
+        "FAQ_HOW_DO_I_CLAIM_MY_REWARDS_QUESTION":"How do I claim my rewards?",
+        "FAQ_HOW_DO_I_CLAIM_MY_REWARDS_ANSWER":"Each block, trading rewards are automatically sent directly to the trader’s dYdX Chain address."
```
Comment (aforaleka) [inline config/localization/en/app.json:1349]: nit: formatting

**Review by aforaleka — CHANGES_REQUESTED** [review]

File: config/localization/en/tooltips.json [hunk 8]
```diff
@@ -198,6 +198,9 @@
     "UNREALIZED_PNL": {
       "TITLE": "Unrealized Profit / Loss",
       "BODY": "Total profit / loss if you were to fully close the remainder of your current position at the current oracle price. Resets if you close your position or change position sides (long to short, or vice versa)."
+    },
+    "REWARD_HISTORY": {
+        "BODY": "Time is displayed in Coordinated Universal Time (UTC)"
```
Comment (mike-dydx) [inline config/localization/en/tooltips.json:203]: note, the tooltip does not have a title per [design](https://www.figma.com/file/fMQodZKzn9B5aZTN5G0fLJ/dYdX-%E2%80%BA-Desktop?type=design&node-id=21144-51979&mode=design&t=7MbRGKHdFcSRqMUS-4)

**Review by aforaleka — APPROVED** [review]

**Pull request merged**